response_operations_ui/templates/create-collection-exercise.html

```diff
@@ -21,76 +23,77 @@ <h2 id="error-creating-ce-message" class="ons-u-fs-r--b">Error creating collecti
                     <p class="ons-u-fs-r">{{ errors }}</p>
                 {% endcall %}
             {% endif %}
-             <h1 class="ons-u-fs-xl">Create collection exercise</h1>
-            <form action="" method="post">
-                {{ form.csrf_token }}
+            <h1 class="ons-u-fs-xl">Create collection exercise | {{survey_ref}} {{short_name}}</h1>
+            <form action="" class="ons-form" method="post">
+                <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+                {% set period_information = '<p>' + 'The period:' + '</p><ul>'
+                    + '<li>' + 'Must not be blank' + '</li>'
+                    + '<li>' + 'Can only be numeric' + '</li>'
+                    + '<li>' + 'Can only be between two and six characters long' + '</li></ul>' %}
+                {% if errors %}
+                    {{
+                        onsPanel({
+                            "body": period_information,
+                        })
+                    }}
+                {% endif %}
+                <label class="ons-label ons-u-fs-m">Period</label>
+                {% if previous_period is not none %}
+                    <span class="ons-label__description">The most recent Period ID added was {{ previous_period }}</span>
+                {% else %}
+                    <span class="ons-label__description">The Period ID is a two, four or six digit number.</span>
```

Comment:
These textboxes need a little bit of padding I think as the text is right up against the input box